packages/odata-common/src/index.ts

```diff
@@ -21,12 +20,21 @@ export {
   OrderableEdmTypeField,
   PropertyMetadata
 } from './selectable';
-export { entityBuilder, EntityBuilderType } from './entity-base';
+export type { ConstructorOrField } from './selectable';
```

Comment:
[q] I don't think it is worth to split this off as the file gets imported anyways, right?

Reply:
Same applies below

Reply:
I guess it's nice as you can read the import statement and already know if it is a type or something else... I am fine with the change, but maybe this is something to mention in our SDK coding style guide (when and when not to use `import type ...`) 
@FrankEssenberger as master of ADRs I kinda hope your responsibility also includes the coding style guide? 😉